website/accounts.py

```diff
@@ -11,10 +11,6 @@
 dispensed with a lot of the validation and niceties one might expect
 in production. Use at your own risk.
 """
-import hashlib
```

Comment:
I also performed a pyflakes pass which eliminated a few unused imports and variables.